compiler/circle-opselector-test/README.md

```diff
@@ -0,0 +1,52 @@
+# Circle-OpSelector-test
+
+## Usage
+
+- This test didn't consider **if** and **while** nodes yet. 
+
+**Single file test**
+
+- `tflite-file-name` must be contained in [tflite receipes](https://github.com/Samsung/ONE/tree/master/res/TensorFlowLiteRecipes)
+- `selected nodes` format is `"0 1 3"`
+
+```
+$ ./opselector_test.sh [tflite-file-name] [selected nodes]
+# ex)
+$ ./compiler/circle-opselector-test/opselector_test.sh Part_Sqrt_Rsqrt_Add_002 "0 1 2 3"
+```
+
+This test compares between the result of `circle-opselector` and [select-operator.py](https://github.com/Samsung/ONE/blob/master/tools/tflitefile_tool/select_operator.py) in tflite_tool.
+
+The detailed process is as follows
```

Comment:
I think this information is implementation detail, it can be moved into `opselector_test.py`.